examples/gno.land/p/demo/grc/grc721/basic_nft.gno

```diff
@@ -324,8 +341,12 @@ func (s *basicNFT) mint(to std.Address, tid TokenID) error {
 	s.balances.Set(to.String(), toBalance)
 	s.owners.Set(string(tid), to)
 
-	event := TransferEvent{zeroAddress, to, tid}
-	emit(&event)
+	std.Emit(
+		MintEvent,
+		"from", "",
```

Comment:
This can be omitted, the `from`

Reply:
65eb0b4aeb310b26ac6bb653e6cffe7e295be485